authorization/role/repository/identity_role.go

```diff
@@ -525,7 +527,8 @@ func (m *GormIdentityRoleRepository) FindScopesByIdentityAndResource(ctx context
 	    FROM
 	      resource
 	    WHERE
-	      resource_id = ? /* RESOURCE_ID */
+          deleted_at IS NULL
```

Comment:
Good catch! gorm does it automatically, whereas in raw queries we gotta do it explicitly.